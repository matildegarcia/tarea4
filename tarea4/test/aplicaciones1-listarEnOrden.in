# se asume un ID máximo de persona de 10

crearTTablaJugadores 20

# Persona jugador5 5, 13 años
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 5 13 Uruguay jugador5
crearFecha 1/12/2022
insertarJugadorEnTabla

# Persona jugador6 6, 19 años
crearJugada
1 4 3 2 1
agregarEnPartida
crearJugador 6 19 Brasil jugador6
crearFecha 5/12/2022
insertarJugadorEnTabla

# Persona jugador3 3, 21 años
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 3 21 Uruguay jugador3
crearFecha 1/1/2023
insertarJugadorEnTabla

# Persona jugador2 2, 7 años
crearJugada
1 2 2 2 2
agregarEnPartida
crearJugador 2 7 Uruguay jugador2
crearFecha 30/11/2022
insertarJugadorEnTabla

# Persona jugador10 10, 15 años
crearJugada
1 3 3 3 3
agregarEnPartida
crearJugador 10 15 Uruguay jugador10
crearFecha 30/10/2022
insertarJugadorEnTabla


# el orden debe ser jugador2, jugador6 y jugador3
listarEnOrden 3 jugador6 jugador3 jugador2

liberarTTablaJugadores

Fin