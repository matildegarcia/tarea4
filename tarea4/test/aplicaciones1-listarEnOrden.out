1>#  se asume un ID máximo de persona de 10.
2>3>#  Persona jugador5 5, 13 años.
4>5>6>7>8>9>#  Persona jugador6 6, 19 años.
10>11>12>13>14>15>#  Persona jugador3 3, 21 años.
16>17>18>19>20>21>#  Persona jugador2 2, 7 años.
22>23>24>25>26>27>#  Persona jugador10 10, 15 años.
28>29>30>31>32>33>#  el orden debe ser jugador2, jugador6 y jugador3.
34>Jugador 2: jugador2, Uruguay, 7 años
Jugada: 1 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
Jugador 6: jugador6, Brasil, 19 años
Jugada: 1 
Movimientos: Abajo Arriba Izquierda Derecha 
Jugador 3: jugador3, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
35>36>Fin.
